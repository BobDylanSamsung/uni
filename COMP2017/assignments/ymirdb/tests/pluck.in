set a 1 2 3
set b 4 5 6 7 8
list entries
pluck a 4
pluck a 3
pluck a 2
pluck a 1
pluck a 0
list entries
pluck b 2
pluck b 3
pluck b 4
list entries
bye
