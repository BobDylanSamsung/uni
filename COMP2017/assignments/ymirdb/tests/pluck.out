ok

ok

b [4 5 6 7 8]
a [1 2 3]

index out of range

3

2

1

index out of range

b [4 5 6 7 8]
a []

5

7

index out of range

b [4 6 8]
a []

bye
